documentation/src/main/asciidoc/upgrading/upgrading.asciidoc

```diff
@@ -7,6 +7,16 @@ The Infinispan community
 
 This guide walks you through the process of upgrading Infinispan.
 
+== Upgrading from 9.0 to 9.1
+
+=== (FineGrained)AtomicMap reimplemented
+
+Infinispan now contains a new implementation of both `AtomicMap` and `FineGrainedAtomicMap`, but the semantics has been preserved. The new implementations don't use `DeltaAware` interface but link:../user_guide/user_guide.html#functional_map_api[Functional API] instead.
```

Comment:
According to http://www.wordreference.com/definition/semantics 'semantics' is used with singular verb, despite it's plural noun.